jsonschema-core.xml

```diff
@@ -1004,6 +1007,22 @@
                     </section>
                 </section>
             </section>
+            <section title="Reserved Locations">
+                <t>
+                    A fourth category of keywords simply reserve a location to hold re-usable
+                    components or data of interest to schema authors that is not suitable
+                    for re-use.  These keywords do not affect validation or annotation results.
```

Comment:
Here I'd suggest to use the term *placeholder keyword* at least once, maybe even emphasized. For example: 

> These *placeholder keywords*  do not affect validation[...] 

Then when the term *placeholder keyword* is used further down below such as in section *References to Possible Non-Schemas* it is terminologically clear that *placeholder keywords*  is the phrase used to refer to keywords falling into the category of *reserved locations*.

Overall the recent additions make things much more consistent and comprehensible, I think. :+1:

Reply:
Oh, I'd actually meant to get rid of "placeholder" and use the "location" terminology.  I'll sort that out and fix the out-of-order wording, too- thanks!